item-spec/common-metadata.md

```diff
@@ -23,14 +23,14 @@ For example, `datetime` is required in STAC Items.
 
 ## Basics
 
-Descriptive fields to give a basic overview of a STAC Item.
+Descriptive fields to give a basic overview of a STAC resource (e.g. Catalog, Collection, Item, Asset).
```

Comment:
I'd prefer the term `entity` or `object` instead of `resource`.

Reply:
```suggestion
Descriptive fields to give a basic overview of a STAC entity (e.g. Catalog, Collection, Item, Asset).
```